### ISSUE TITLE
Tooltip: Hoverable Tooltip correct behaviour

### ISSUE THREAD

**Issue by marcoscv-work** [liferay/clay#4976]
We should be able to move the mouse pointer over the tooltip, right now it is impossible because when we leave the trigger element the tooltip is hidden.

<img width="144" alt="image" src="https://user-images.githubusercontent.com/7963804/177566545-47892d56-ffaf-4f8f-9783-13bf575a369d.png">

In order to accomplish success criterion [1.4.13 Content on Hover or Focus](https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus) (Level AA) we should add the possibility to move the mouse over the tooltip.

**Links / references:**

https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html#example-2-hoverable-tooltip

**Hoverable Tooltip**
![image](https://user-images.githubusercontent.com/7963804/177567350-a3e244b0-e53e-4b0e-82db-d4389315aa4e.png) 
![image](https://user-images.githubusercontent.com/7963804/177567402-c4ea24e2-54d9-4a07-813f-006a1711f9a4.png)

_Screenshot of a button with a large mouse pointer over it, and a tooltip displayed  below the button which is obscured by the large pointer Screenshot of a button with a tooltip below it, and a large mouse pointer at the bottom of the tooltip
Figure 3 A button's tooltip is displayed directly below it on mouse hover which can easily be obscured by a large pointer. The tooltip itself is able to be hovered so the mouse pointer can be moved down to its bottom edge in order to view the tooltip text._

**Example of an accessibility correct implementation**:
https://www.w3.org/WAI/WCAG21/Techniques/client-side-script/SCR39
https://www.w3.org/WAI/WCAG21/working-examples/script-hoverable/

**Comment by drakonux** [comment]
cc\ @liferay-lexicon
